osu.Game/Graphics/UserInterface/Nub.cs

```diff
@@ -149,7 +149,7 @@ public Color4 GlowColour
                 glowColour = value;
 
                 var effect = EdgeEffect;
-                effect.Colour = value;
+                effect.Colour = value.Opacity(0);
```

Comment:
Just to make sense of things:
```suggestion
                effect.Colour = Glowing ? value : value.Opacity(0);
```